services/repository/contributors_graph.go

```diff
@@ -150,6 +151,30 @@ func getExtendedCommitStats(repo *git.Repository, revision string /*, limit int
 				authorEmail := strings.TrimSpace(scanner.Text())
 				scanner.Scan()
 				date := strings.TrimSpace(scanner.Text())
+
+				var coAuthors []*api.CommitUser
+				emailSet := map[string]bool{}
+				for scanner.Scan() {
+					line := scanner.Text()
+					if line == "" {
+						// There should be an empty line before we read the commit stats line.
+						break
+					}
+					coAuthorName, coAuthorEmail, err := util.ParseCommitTrailerValueWithAuthor(line)
+					if err != nil {
+						continue
```

Comment:
It's better to have a log here?

Reply:
I'm not sure how frequent it is for commits to have invalid `Co-authored-by` trailers but if it won't cause a problem for admins we could have this as maybe an INFO/DEBUG level log